tests/attestation_plugin/host/host.c

```diff
@@ -116,6 +161,33 @@ int main(int argc, const char* argv[])
         argv[1], OE_ENCLAVE_TYPE_AUTO, flags, NULL, 0, &enclave);
     OE_TEST(result == OE_OK);
 
+    /*
+     * To test sgx pccs lib v3.0 and v3.1, we read evidence and endorsements
+     * from file.
+     * This test can be removed when the testing pipelines are updated
+     * to use the latest sgx pccs lib.
+     */
+
+    size_t evidence_size, endorsements_size;
+    uint8_t* evidence = NULL;
+    uint8_t* endorsements = NULL;
+
+    read_file_to_buffer(&evidence, &evidence_size, "evidence.txt", argv[2]);
+    read_file_to_buffer(
+        &endorsements, &endorsements_size, "endorsements.txt", argv[2]);
+
+    if (endorsements && evidence)
```

Comment:
Based on the current code, I cannot see how endorsements and evidence can be NULL. Please handle the error conditions properly even if this is a test, which helps when someone need to debug the test.